packages/terra-slide-panel-manager/src/terra-dev-site/doc/example/SlidePanelManagerSquishExample.jsx

```diff
@@ -275,9 +276,7 @@ ContentComponent.contextType = DisclosureManagerContext;
 
 const SlidePanelManagerExample = () => (
   <div className={cx('example-wrapper')}>
-    <SlidePanelManager
-      panelBehavior="squish"
-    >
```

Comment:
This was an optional change right? You didn't have to make this change for the slide panel manager to continue working?

Reply:
@mjhenkes Yes it's an optional change. Now that we support `typeConfig` it made sense to [update](https://github.com/cerner/terra-framework/pull/1179/files/4d5bef74cd7f17b16468058985b2ef1b0703de7a#diff-5c42752b6d6b7c181b39bd537ef41850R189) the example.